scripts/mutect2_wdl/mutect2_nio.wdl

```diff
@@ -0,0 +1,972 @@
+## scatter_count: number of parallel jobs to generate when scattering over intervals
+## artifact_modes: types of artifacts to consider in the orientation bias filter (optional)
+## m2_extra_args, m2_extra_filtering_args: additional arguments for Mutect2 calling and filtering (optional)
+## split_intervals_extra_args: additional arguments for splitting intervals before scattering (optional)
+## run_orientation_bias_filter: if true, run the orientation bias filter post-processing step (optional, false by default)
+## run_oncotator: if true, annotate the M2 VCFs using oncotator (to produce a TCGA MAF).  Important:  This requires a
+##                   docker image and should  not be run in environments where docker is unavailable (e.g. SGE cluster on
+##                   a Broad on-prem VM).  Access to docker hub is also required, since the task downloads a public docker image.
+##                   (optional, false by default)
+##
+## ** Primary inputs **
+## ref_fasta, ref_fai, ref_dict: reference genome, index, and dictionary
+## tumor_bam, tumor_bam_index: BAM and index for the tumor sample
+## normal_bam, normal_bam_index: BAM and index for the normal sample
+##
+## ** Primary resources ** (optional but strongly recommended)
+## pon: optional panel of normals in VCF format containing probable technical artifacts (false positves)
+## gnomad: optional database of known germline variants (see http://gnomad.broadinstitute.org/downloads)
+## variants_for_contamination: VCF of common variants with allele frequencies for calculating contamination
+##
+## ** Secondary resources ** (for optional tasks)
+## onco_ds_tar_gz, default_config_file: Oncotator datasources and config file
+## sequencing_center, sequence_source: metadata for Oncotator
+## filter_oncotator_maf: Whether the MAF generated by oncotator should have the filtered variants removed. Default: true
+##
+## Outputs :
+## - One VCF file and its index with primary filtering applied; secondary filtering and functional annotation if requested; a bamout.bam
+##   file of reassembled reads if requested
+##
+## Cromwell version support
+## - Successfully tested on v30
+##
+## LICENSING :
+## This script is released under the WDL source code license (BSD-3) (see LICENSE in
+## https://github.com/broadinstitute/wdl). Note however that the programs it calls may
+## be subject to different licenses. Users are responsible for checking that they are
+## authorized to run all programs before running this script. Please see the docker
+## pages at https://hub.docker.com/r/broadinstitute/* for detailed licensing information
+## pertaining to the included programs.
+workflow Mutect2 {
+    # Mutect2 inputs
+    File? intervals
+    File ref_fasta
+    File ref_fai
+    File ref_dict
+    File tumor_bam
+    File tumor_bai
+    File? normal_bam
+    File? normal_bai
+    File? pon
+    Int scatter_count
+    File? gnomad
+    File? variants_for_contamination
```

Comment:
Why eliminate all the index files?  Seems like that will make it harder to keep the files in sync.  Your call.